book/07-git-tools/sections/rerere.asc

```diff
@@ -1,31 +1,31 @@
 [[_rerere]]
-=== Rerere
+=== Ang Rerere
 
-The `git rerere` functionality is a bit of a hidden feature.
-The name stands for ``reuse recorded resolution'' and, as the name implies, it allows you to ask Git to remember how you've resolved a hunk conflict so that the next time it sees the same conflict, Git can resolve it for you automatically.
+Ang `git rerere` pag-andar ay maliit na nakatagong tampok.
+Ang pangalan ay nakatayong ``reuse recorded resolution'' at, bilang nagpapahiwatig na pangalan, pinapayagan ka nitong hilingan ang Git na tandaan kung paano mo nalutas ang isang malaking naputol na pagsasalungat kaya sa susunod na makakita ito ng parehong pagkasalungat, ang Git ay maaaring malutas ito para sa iyo ng awtomatiko.
 
-There are a number of scenarios in which this functionality might be really handy.
-One of the examples that is mentioned in the documentation is when you want to make sure a long-lived topic branch will ultimately merge cleanly, but you don't want to have a bunch of intermediate merge commits cluttering up your commit history.
-With `rerere` enabled, you can attempt the occasional merge, resolve the conflicts, then back out of the merge.
-If you do this continuously, then the final merge should be easy because `rerere` can just do everything for you automatically.
+Mayroong mga bilang ng mga sitwasyon na kung saan ang pag-andar nito ay maaaring talagang magaling.
+Isa sa mga halimbawa na nabanggit sa dokumentasyon ay kapag gusto mo na tiyakin ang isang matagal na paksa na branch ay ganap na pagsasama nang malinis, ngunit ayaw mong makakaroon ng isang grupo ng intermediate merge na mga commit na nag-clutter pataas sa iyong kasaysayan ng commit.
+Sa `rerere` na pinagana, maaari kang magtangka sa minsang na-merge, malutas ang mga pagkasalungat, pagkatapos ay i-atras ang pag-merge.
+Kung gagawin mo ito nang paulit-ulit, pagkatapos ay ang huling merge ay dapat na madali dahil sa `rerere` ay maaari lamang gawin ang lahat para sa iyo ng awtomatiko.
 
-This same tactic can be used if you want to keep a branch rebased so you don't have to deal with the same rebasing conflicts each time you do it.
-Or if you want to take a branch that you merged and fixed a bunch of conflicts and then decide to rebase it instead -- you likely won't have to do all the same conflicts again.
+Ang parehong taktika ay maaaring gamitin kung gusto mong itago ang isang branch na naka-rebase rebased kaya hindi mo kailangang harapin ang parehong pag-rebase sa mga sumasalungat sa bawat oras na ginawa mo ito.
+O kung gusto mong kumuha ng isang branch na  if you want to take a branch na nai-merge mo at nag-ayos ng maraming mga pagkakasalugat at pagkatapos ay magpasya na i-rebase ito sa halip -- malamang na hindi mo kailanganin gawin muli ang parehong mga pagkakasalungat muli.
```

Comment:
please remove the source phrase
~~if you want to take a branch~~

Reply:
done change